tests/test_dates.py

```diff
@@ -155,6 +160,28 @@ def test_local_day_of_week_standalone(self):
         fmt = dates.DateTimeFormat(d, locale='bn_BD')
         self.assertEqual('4', fmt['c']) # friday is first day of week
 
+    def test_pattern_day_of_week(self):
+        dt = datetime(2016,2,6)
+        fmt = dates.DateTimeFormat(dt, locale='en_US')
```

Comment:
I'd love to see the "local day of week" formats (`e` and `c`) being tested with a locale that has another start-of-week-day than `en_US`'s Sunday.

Otherwise, this PR looks very good!